.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["unbubbled"]
+}
```

Comment:
Well, the template explicitly makes an exception for settings as it may contain things that should be shared by developers, such as subdirectories for a project.

In this case, a config for cSpell would IMO make sense as well, however such word as `unbubbled` doesn't exist, so the warings seem valid and a rename would help.

Reply:
The name has been changed to from `unbubbledToggle` to `notBubblingToggle` so the problem has vanished